src/plugins/intel_gpu/src/graph/primitive_inst.cpp

```diff
@@ -996,6 +1004,14 @@ void primitive_inst::update_paddings() {
             reset_pad(*_impl_params, _node);
         return;
     }
+    // Reset paddings used in the previous iteration for crop before executing do_runtime_in_place_crop
+    for (auto u : get_user_insts()) {
+        if (u->get_node().is_type<crop>() && u->_impl_params->output_layouts[0].data_padding.get_dynamic_pad_dims() != tensor(0)) {
+            if (u->get_node().can_be_optimized()) {
+                reset_pad(*u->_impl_params, u->_node);
+            }
```

Comment:
Why should we do this at the previous node of crop? (i.e., why not when current node is crop?)

Reply:
During `execute`, it's executed in the following order: `update_shape` -> `update_paddings` -> `do_runtime_in_place_crop`. Therefore, in `do_runtime_in_place_crop` of previous node, `FC`, the padding of the output layout of the user(`Crop`) has already been updated in advance. So if check whether the current node is crop or not in `Crop`'s `update_paddings`, the updated padding will be reset again. Therefore, logic for searching user and reset padding is needed here.